core-bundle/src/Resources/contao/modules/ModuleSearch.php

```diff
@@ -284,7 +284,7 @@ protected function compile()
 				$objTemplate->url = StringUtil::specialchars(urldecode($arrResult[$i]['url']), true, true);
 				$objTemplate->title = StringUtil::specialchars(StringUtil::stripInsertTags($arrResult[$i]['title']));
 				$objTemplate->class = (($i == ($from - 1)) ? 'first ' : '') . (($i == ($to - 1) || $i == ($count - 1)) ? 'last ' : '') . (($i % 2 == 0) ? 'even' : 'odd');
-				$objTemplate->relevance = sprintf($GLOBALS['TL_LANG']['MSC']['relevance'], number_format($arrResult[$i]['relevance'] / $arrResult[0]['relevance'] * 100, 2) . '%');
+				$objTemplate->relevance = sprintf($GLOBALS['TL_LANG']['MSC']['relevance'], number_format(($arrResult[$i]['relevance'] ?: 0) / ($arrResult[0]['relevance'] ?: 1) * 100, 2) . '%');
```

Comment:
Not sure if we shouldn't do that in `Search::searchFor()` for BC reasons?

Reply:
Done in debb634373d86c967ab2cd963a2abdf296fd4924